requirements-test.txt

```diff
@@ -0,0 +1,8 @@
+coverage==4.4.1
+mock>=1.0.1
+flake8>=2.1.0
+tox>=1.7.0
+codecov>=2.0.0
```

Comment:
can you please remove `flake8` and add `openwisp-utils[qa]` (which includes flake8 and isort) as here?
https://github.com/openwisp/django-netjsonconfig/blob/master/requirements-test.txt#L3